client/src/components/PublicComment/postPublicComment.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
```

Comment:
this is a clinet-side servce, which we normally put in the /src/services folder.